index.html

```diff
@@ -1,10 +1,75 @@
 <!DOCTYPE html>
 <html>
     <head>
-    <title></title>
+    <title>TALENT SHOW</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="utf-8"> 
+    <META name="keywords" lang="en-us" content="events,plot,night_life">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+     
     </head>
     <body>
+            <div class="topnav" id="myTopnav">
+                    <a href="#home" class="active">Home</a>
+                    <a href="#news">News</a>
+                    <a href="#contact">Contact</a>
+                    <a href="#about">About</a>
+                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                      <i class="fa fa-bars"></i>
+                    <a href="#" class="right">login</a>
+                  </div>
+                  
+                  <div class="container">
+                      <h2>WHITE TALENT SHOW</h2>
+                      <img SRC="images/img.jpg" class="" alt="">
+                  
+                  
 
+                      <div class="col-md-4">
+                          <form action="post">
+                              <div class="group">
+                              <label>Name</label>
+                              <br>
```

Comment:
yesterday they gave us a recommendation not to use "<br>"

Reply:
can u instead use divs

Reply:
thanks